releasenotes/notes/add-process-matrix-plot-12f9456fcd47be52.yaml

```diff
@@ -0,0 +1,28 @@
+---
+features:
+  - |
+    Add ``plot_quantum_channel_city`` function to visualization module.
+    This function visualize Chi-matrix given by arbitrary quantum channel object
+    in a cityscape fashion, for example::
+
+        from qiskit import QuantumCircuit
+        from qiskit import quantum_info as qi
+        from qiskit.visualization import plot_process_city
+
+        qc = QuantumCircuit(2)
+        qc.cx(0, 1)
+
+        plot_quantum_channel_city(qc)
+  - |
+    Add ``plot_quantum_channel_hinton`` function to visualization module.
+    This function visualize Chi-matrix given by arbitrary quantum channel object
+    as a Hinton diagram, for example::
+
+        from qiskit import QuantumCircuit
```

Comment:
These examples should be in the docstrings as well.

Reply:
it's already in here
https://github.com/Qiskit/qiskit-terra/blob/45acb2d86de338ffb84aa52064e80662ad4d3a33/qiskit/visualization/state_visualization.py#L152-L163